### PR TITLE
Remove unnecessary log formatting

### PR DESCRIPTION
**Pull request by seallard**
## Description
I don't like this way of discarding all context for an exception, super annoying. Sure the logs look prettier but they also become useless.

### Fixed
- log formatting

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: trailblazer/io/controller.py [hunk 1]
```diff
@@ -18,10 +18,7 @@ class ReadFile:
     @classmethod
     def get_content_from_file(cls, file_format: str, file_path: Path) -> Any:
         """Read file using file format dispatch table."""
-        try:
-            return cls.read_file[file_format](file_path=file_path)
-        except FileNotFoundError as error:
-            raise FileNotFoundError(f"File not found: {file_path}") from error
+        return cls.read_file[file_format](file_path=file_path)
```
Comment (henrikstranneheim) [inline trailblazer/io/controller.py:21]: Won't this risk stopping a TB scan if a file is missing?

Comment (seallard) [reply trailblazer/io/controller.py:21]: No, the catch all below remains! I added this line in my last PR

Comment (henrikstranneheim) [reply trailblazer/io/controller.py:21]: 👍

File: trailblazer/store/crud/update.py [hunk 2]
```diff
@@ -154,9 +154,7 @@ def update_analysis_from_slurm_output(
                 analysis=analysis, analysis_host=analysis_host
             )
         except Exception as exception:
-            LOG.error(
-                f"Error updating analysis for: case - {analysis.case_id} : {exception.__class__.__name__}"
-            )
+            LOG.error(f"Error updating analysis for: case - {analysis.case_id} : {exception}")
```
Comment (henrikstranneheim) [inline trailblazer/store/crud/update.py:157]: 👍

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**